e2e/tests-dfx/install.bash

```diff
@@ -81,3 +81,54 @@ teardown() {
     dfx canister create --all
     assert_command_fail dfx canister install --all --wasm "${archive:?}/wallet/0.10.0/wallet.wasm"
 }
+
+@test "install runs post-install tasks" {
+    install_asset post_install
+    dfx_start
+
+    assert_command dfx canister create --all
+    assert_command dfx build
+
+    assert_command dfx canister install postinstall
+    assert_match hello
+
+    assert_command dfx canister install postinstall_script
+    assert_match hello
+    
+    echo 'return 1' >> postinstall.sh
+    assert_command_fail dfx canister install postinstall_script --mode upgrade
+    assert_match hello
+}
+
+@test "post-install tasks receive environment variables" {
+    install_asset post_install
+    dfx_start
+    echo "echo \$CANISTER_ID" >> postinstall.sh
+
+    assert_command dfx canister create --all
+    assert_command dfx build
+    id=$(dfx canister id postinstall_script)
+
+    assert_command dfx canister install --all
+    assert_match "hello $id"
```

Comment:
Can you make the output for the two canisters different?  Here we expect both post-install scripts to execute, but can't tell the difference.